pkg/api/handlers/generic/images.go

```diff
@@ -305,7 +305,7 @@ func GetImages(w http.ResponseWriter, r *http.Request) {
 		utils.Error(w, "Something went wrong.", http.StatusInternalServerError, errors.Wrap(err, "Failed get images"))
 		return
 	}
-	var summaries = make([]*handlers.ImageSummary, len(images)+1)
+	var summaries = make([]*handlers.ImageSummary, len(images))
```

Comment:
The +1 ensures we don't return an empty body. When no images are found.

Reply:
just spun up an f31 vm with moby-engine, with no images pulled down a curl to /images/json returns and empty array "[]", on podman in the same state if i put the +1 back in the a curl to /images/json returns "[null]" which is causing consuming code to fail.  without the +1 podman produced the same as docker "[]".

without the +1 podman mimics the behavior of docker on this call

Reply:
@st1971 Okay, something must have changed in the response writer.  Sorry for the noise, thanks for checking.

Reply:
no worries